operator/api/loki/v1/lokistack_types.go

```diff
@@ -837,12 +837,12 @@ type OTLPSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Stream Labels"
 	StreamLabels *OTLPStreamLabelSpec `json:"streamLabels,omitempty"`
 
-	// StructuredMetadata configures which attributes are saved in structured metadata.
+	// Drop configures which attributes are dropped from the log entry.
```

Comment:
Should we mention here the detail on how the attributes should be in their OTEL formatted name and not Loki?

Reply:
I wondered about how to do that. Let me try adding some text to the `OTLPSpec`, because this is the same for both `StreamLabels` and `Drop`.